src/components/beneficiarySelectionContent/beneficiarySelectionContent.tsx

```
@@ -14,7 +14,7 @@ import { setBeneficiaries as setBeneficiariesAction } from '../../redux/actions/
 import { toastNotification } from '../../redux/actions/uiAction';
 import { DEFAULT_USER_DRAFT_ID } from '../../redux/constants/constants';
 import { Beneficiary } from '../../redux/reducers/editorReducer';
-import { isThreeSpeakBeneficiary } from '../../providers/speak/beneficiary';
+import { isThreeSpeakBeneficiary } from '@ecency/sdk';
```

ESLint

[error] `@ecency/sdk` import should occur before import of `./styles`

(import/order)